src/static/javascripts/components/questions.jsx

```diff
@@ -186,6 +194,7 @@ var FreeResponse = React.createClass({
 });
 /*
 *Rating slider
+*This still needs work from Michael and I
```

Comment:
@MCChung303 let me know when you want to work on this, I saved it so I can show you handler stuff.